docs/samples/contact-center/app.js

```diff
@@ -293,6 +299,10 @@ incomingCallListener.addEventListener('task:incoming', (event) => {
 function answer() {
   answerElm.disabled = true;
   declineElm.disabled = true;
+  holdElm.disabled = false;
```

Comment:
Had to make this change here as I was not getting the other event even after accepting the call.
If this is wrong, will change it.